zkevm-circuits/src/bytecode_circuit.rs

```diff
@@ -576,19 +574,17 @@ impl<F: Field> BytecodeCircuitConfig<F> {
     pub(crate) fn assign_internal(
         &self,
         layouter: &mut impl Layouter<F>,
-        size: usize,
+        degree: u32,
+        unusable_rows: usize,
         witness: &BytecodeCircuitAssignment<F>,
         challenges: &Challenges<Value<F>>,
     ) -> Result<(), Error> {
-        // Subtract the unusable rows from the size
```

Comment:
Nitpick: notice `unusable_rows` and `degree` is just to compute `last_row_offset`. Do you think is it good idea to compute `last_row_offset` outside and just pass `last_row_offset` as assign_internal argument?